apps/frontend/src/app/about/features/features.json

```diff
@@ -171,6 +171,17 @@
         "Calendly": true
       }
     },
+    {
```

Comment:
I would split this in two seperate features. I know they are closely related, but they are still something different.

Reply:
Can do but won't check the other services for now xD